brave/browser-laptop
# Fix helpful hint wording

Test Plan:
1. Open Brave Preferences.
2. Look at the helpful hints in the bottom left corner.
3. If the hint is not about HTTPS Everywhere, click the refresh button until you find it.
4. Make sure the wording makes sense.

Fixes #4968
